Application will not load in android 11 when trying to use suggested sdk.

In Android when trying to do the suggested implementation. the application gets crashed on load for android 11. works fine with all other versions of android

are you getting the error:

```
    java.lang.ExceptionInInitializerError
        at okhttp3.internal.platform.Platform.get(Platform.java:85)
        at okhttp3.OkHttpClient.newSslSocketFactory(OkHttpClient.java:263)
        at okhttp3.OkHttpClient.<init>(OkHttpClient.java:229)
        at okhttp3.OkHttpClient$Builder.build(OkHttpClient.java:1015)
        at com.google.maps.OkHttpRequestHandler$Builder.build(OkHttpRequestHandler.java:201)
        at com.google.maps.GeoApiContext$Builder.build(GeoApiContext.java:625)
        (...)
     Caused by: java.lang.IllegalStateException: Expected Android API level 21+ but was 30

```

--- a/build.gradle
+++ b/build.gradle
@@ -52,13 +52,13 @@ artifacts {
 
 dependencies {
     compileOnly 'com.google.appengine:appengine-api-1.0-sdk:1.9.76'
-    api 'com.squareup.okhttp3:okhttp:3.14.4'
+    api 'com.squareup.okhttp3:okhttp:4.9.1'
     api 'com.google.code.gson:gson:2.8.6'
     api 'io.opencensus:opencensus-api:0.25.0'
     implementation 'org.slf4j:slf4j-api:1.7.26'
     testImplementation 'junit:junit:4.12'
     testImplementation 'org.mockito:mockito-core:3.0.0'
-    testImplementation 'com.squareup.okhttp3:mockwebserver:3.14.2'
+    testImplementation 'com.squareup.okhttp3:mockwebserver:4.9.1'
     testImplementation 'org.apache.httpcomponents:httpclient:4.5.9'
     testImplementation 'org.slf4j:slf4j-simple:1.7.26'
     testImplementation 'org.apache.commons:commons-lang3:3.9'

